zsign.cpp

```diff
@@ -64,6 +65,31 @@ int usage()
 	return -1;
 }
 
+string getCacheDirectory() {
+    try {
+#ifdef _WIN32
+        const char* buffer = std::GetEnvironmentVariable("LOCALAPPDATA", nullptr, 0);
+		string cacheDir = std::filesystem::path(buffer) / "zsign_cache";
```

Comment:
This line does not seem to be at the proper indentation level (and/or uses tabs/spaces inconsistently).